File: Python/Reverse-Of-A-Number.py
# Question : To reverse a given number 
#        Input : User input a number
#        Output : Reverse of the given number

n=int(input("Enter the number to reverse : "))
rev=0
while(n>0):
    rev=(rev*10)+n%10
    n=n//10
print("Rverse  of the given number is : " ,rev)

File: C/Reverse-of-A-Number.c
/* Question : To reverse a given number 
        Input : User input a number
        Output : Reverse of the given number */
#include<stdio.h>
int main()
{
    int n, reverse=0, rem;    
printf("Enter a number: ");    
  scanf("%d", &n);    
  while(n!=0)    
  {    
     rem=n%10;    
     reverse=reverse*10+rem;    
     n/=10;    
  }    
  printf("Reversed Number: %d",reverse);    
    return 0;
}
